packages/css-library/dist/tokens/css/variables.css

```diff
@@ -135,6 +135,8 @@
   --vads-font-line-height-body-lead: 1.75;
   --vads-font-line-height-default: 1.5;
   --vads-font-line-height-heading: 1.2;
+  --vads-font-size-source-sans-normalized: 1.06rem;
+  --vads-font-size-root: 1rem;
```

Comment:
@powellkerry @micahchiang If we're renaming `font-size-base` to `font-size-root`, then anywhere we use this token will need to be updated for the name change. If that's the case, why wouldn't we point to the newly added `vads-font-size-root` instead? They both have the same value/meaning, right? What are the reasons we need to have both these tokens?

Reply:
I was wondering about that but thought I would leave it until someone validated my thoughts :) The --vads--font-size-root should be removed now.

Reply:
@powellkerry @danbrady - I added a comment below. We can keep the css custom property that gets generated - `font-size-root` - because the original, `font-size-base`, wasn't being used anywhere in vets-website.